### ISSUE TITLE
Update to GoGo Loco v1.8.6

### ISSUE THREAD

**Issue by KuryKat** [LastationVRChat/NSFW-GoLoco#14]
### The project version that you checked that didn't have the feature you are asking for

v1.8.3-NSFW-1.1.0

### Which Unity Version are you using?

2022.3.22f1

### Describe your suggestion

GoGo Loco recently updated to v1.8.6, skipping from v1.8.3 to v1.8.5, and introduced several new features and fixes, including improved dash controls, modular avatar support, seated emotes outside of stations. To keep NSFW Locomotion up to date and take advantage of these improvements, I suggest upgrading our version to match the latest GoGo Loco release.

### Why would you like this added/changed?

Staying current with GoGo Loco's updates will ensure we benefit from the latest features, fixes, and optimizations, improving both functionality and user experience. The new features, such as seated emotes, dash gimmicks, and modular avatar support, could provide more flexibility and stability for users.

**Comment by TheSlowlyy** [comment]
Just checking -- is there an update to this?

**Comment by KuryKat** [comment]
> Just checking -- is there an update to this?

Yes, it's in the works, but it has been a lower priority compared to other ongoing projects. Given our limited resources, we have to balance development efforts carefully. We'll provide updates when there's progress to share.

**Comment by Shana195** [comment]
any luck so far with the update?

**Comment by ultraspacemommy** [comment]
i don't mean to be a bother but any update on the progress

**Comment by PineCone69420** [comment]
I don't know how difficult this will be, but I may take a crack at updating this to the latest GoGo since it seems like development has stalled.

I assume its as easy as just dropping in the animations but probably not. Regardless I really do want to see this use the latest GoGo so I'll still take a crack at it.

I am going to be moving and unable to access my PC in about a month so... if there are no updates after a few months, I probably gave up or I would have not had the ability to work on this. :P
